customer/src/main/java/no/unit/nva/customer/get/GetCustomerHandler.java

```diff
@@ -90,8 +61,22 @@ protected UUID getIdentifier(RequestInfo requestInfo) throws ApiGatewayException
         return Constants.DEFAULT_RESPONSE_MEDIA_TYPES;
     }
 
+    @Override
+    protected CustomerDto processInput(Void input, RequestInfo requestInfo, Context context)
+        throws ApiGatewayException {
+        return customerService.getCustomer(getIdentifier(requestInfo));
+    }
+
     @Override
     protected Integer getSuccessStatusCode(Void input, CustomerDto output) {
         return SC_OK;
     }
+
+    @JacocoGenerated
+    private static DynamoDBCustomerService defaultCustomerService() {
```

Comment:
defaultCustomerService method is duplicated in many classes, but now it can be defined once and re-used, in another PR.